src/sidebars/favourites/tearout.js

```diff
@@ -11,37 +11,53 @@
                         var dragElement = element[0],
                             tearElement = dragElement.parentNode.parentNode,
                             tileWidth = tearElement.clientWidth,
-                            tileHeight = tearElement.clientHeight;
+                            tileHeight = tearElement.clientHeight,
+                            store;
 
-                        function createConfig() {
+                        function createConfig(minWidth, minHeight, width, height, url, resizable, maximizable, showTaskbarIcon, saveWindowState) {
```

Comment:
That's fair enough. It's only called in 2 places, if it were called more frequently and in more configurations it might be worth it